### PR TITLE
Feat/login

### PR DESCRIPTION
**Pull request by phong9xNcc**
- login Ui and feat

### CODE REVIEW

File: libs/utils/src/lib/validate/index.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// import * as yup from 'yup';
```
Comment (minhlucncc) [inline libs/utils/src/lib/validate/index.ts:1]: remove comment, or remove the whole file

File: apps/chat/src/app/pages/login/index.tsx [hunk 2]
```diff
@@ -1,37 +1,48 @@
...
+          {
+            background:
+              'linear-gradient(219.23deg, #2970FF 1.49%, #8E84FF 43.14%, #E0D1FF 94.04%)',
+          } as any
+        }
+      >
+        <div className="flex-row justify-center items-center flex w-[850px] h-fit p-12 gap-x-12 rounded-2xl bg-[#0b0b0b]">
+          <div className="flex-col justify-start items-center flex w-full h-fit p-0 gap-y-8">
+            <TitleSection />
+            <GoogleButtonLogin />
```
Comment (minhlucncc) [inline apps/chat/src/app/pages/login/index.tsx:20]: adding callback for `GoogleButtonLogin` to control login flow

File: apps/chat/src/app/pages/login/index.tsx [hunk 3]
```diff
@@ -1,37 +1,48 @@
...
-                        <LoginForm onSubmit={handleSubmit} />
-                    </div>
-                </div>
-            </div>
-        </section>
-    );
+  return (
+    <>
+      <div
+        className=" w-screen h-screen flex items-center justify-center"
```
Comment (minhlucncc) [inline apps/chat/src/app/pages/login/index.tsx:9]: split login block to a new compomnent

File: apps/chat/src/app/mock/state.ts [hunk 4]
```diff
@@ -5,9 +5,13 @@ const preloadedState: PreloadedRootState = {
       theme: 'light',
       loadingStatus: 'loaded', 
     },
-    auth: {
-      loadingStatus: 'loaded',
-    },
+    // auth: {
```
Comment (minhlucncc) [inline apps/chat/src/app/mock/state.ts]: remove

File: libs/utils/src/lib/validate/index.ts (hunk 1 above)
Comment (phong9xNcc) [reply libs/utils/src/lib/validate/index.ts:1]: done e

File: apps/chat/src/app/pages/login/index.tsx (hunk 2 above)
Comment (phong9xNcc) [reply apps/chat/src/app/pages/login/index.tsx:20]: done e

File: apps/chat/src/app/mock/state.ts (hunk 4 above)
Comment (phong9xNcc) [reply apps/chat/src/app/mock/state.ts]: done

**Pull request merged**